gwardia_hub/settings.py

```diff
@@ -112,9 +113,7 @@
 # https://docs.djangoproject.com/en/5.0/howto/static-files/
 
 STATIC_URL = "static/"
-STATICFILES_DIRS = [
-    BASE_DIR / "static/GwardiaHub/"
-]
+STATICFILES_DIRS = [BASE_DIR / "static/GwardiaHub/"]
```

Comment:
Why did you move this to one line?

Reply:
If it fits in one line then why not?

Reply:
honesly i dont remember doing that but i guess it doesnt matter

Reply:
PRs and commits should change only things related to this. Probably Ruff/Pycharm formatter cleaned up this code... So the problem is with previous PR where someone didn't format it correctly.